src/RpcServer/RpcServerSettings.cs

```diff
@@ -19,9 +20,32 @@ internal class Settings
         public string[] DisabledMethods { get; }
         public int MaxConcurrentConnections { get; }
 
-        public static Settings Default { get; private set; }
+        public RpcServerSettings(IPAddress bindAddress = null,
```

Comment:
I am not very fan of constructors with all the arguments, I prefer to set the properties manually with the `get` and `set`

Reply:
I designed it this way so that the properties would continue to be read only, but I'm fine with changing it. I will point out that all the ctor args have defaults, which makes it easy to specify only the non-default fields via named arguments.

``` csharp
// all the fields except port use the defaults
var rpcSettings = new RpcServerSettings(port: rpcPort);
```

Reply:
Actually, it would make sense to move to R/W properties in order to support auto-reconfiguration - at least for the settings that can be modified while running (RpcUser, RpcPass, MaxGasInvoke, MaxFee, DisabledMethods)